syftlib/src/main/java/org/openmined/syft/proto/Placeholder.kt

```diff
@@ -9,17 +9,18 @@ data class Placeholder(
     val description: String
 ) {
     companion object {
+        // Generate Placeholder object by parsing PlaceholderOuterClass.Placeholder object
```

Comment:
may be also add the class description like 
- Placeholder class is used to reference the order of the tensors in the plan.